btree/btree.c

```diff
@@ -314,17 +313,17 @@
  * Iteration function will look for a key and read keys one-by-one, if returned key is last key in the node , we may need to fetch next node,
  * To fetch keys in next node: first release the LOCK, call Iteration function and pass key=last fetched key and next_sibling_flag=1
  * If next_sibling_flag == 1, we will also load next sibling and , we need to fetch key next to given key
- * As we are releasing lock for finding next node, updates such as(insertion of new keys, merging due to deletion can happened), 
- * so in such cases, we will load both earlier node and node next to it
+ * As we are releasing lock for finding next node, updates such as(insertion of new keys, merging due to deletion) can happen, 
+ * so in such cases, we will load both earlier node and node to the right of it
  * 
  * Phases Description: 
- * step 1. NEXTDOWN: while loading next node, it will traverse and load leftmost child, 
- *                   else it will traverse and load nodes by searching for given key. 
+ * step 1. NEXTDOWN: this state will load nodes searching by the given key if next_sibling_flag == 0, 
+ *                   but if next_sibling_flag == 1 then this state will load the leftmost child nodes during its downward traversal.
```

Comment:
Change
this state will load nodes searching by the given key if next_sibling_flag == 0, 
                    but if next_sibling_flag == 1 then this state will load the leftmost child nodes during its downward traversal.
to
this state will load nodes searching for the given key,
                    but if next_sibling_flag == 'True' then this state will also load the leftmost child nodes during its downward traversal.